camera_calibration/src/camera_calibration/camera_calibrator.py

```diff
@@ -186,24 +231,37 @@ def handle_monocular(self, msg):
 
         # This should just call the MonoCalibrator
         drawable = self.c.handle_msg(msg)
-        self.displaywidth = drawable.scrib.shape[1]
-        self.redraw_monocular(drawable)
+        if not self._no_gui: 
+            self.displaywidth = drawable.scrib.shape[1]
+            self.redraw_monocular(drawable)
+        else:
+            self.lpub.publish(self.bridge.cv2_to_imgmsg(drawable.scrib, "bgr8"))
+            if drawable.good_sample:
+                self.queue_terminal.put(True)
 
     def handle_stereo(self, msg):
         if self.c == None:
             if self._camera_name:
                 self.c = StereoCalibrator(self._boards, self._calib_flags, self._fisheye_calib_flags, self._pattern, name=self._camera_name,
                                           checkerboard_flags=self._checkerboard_flags,
-                                          max_chessboard_speed = self._max_chessboard_speed)
+                                          max_chessboard_speed = self._max_chessboard_speed, no_mono_if_stereo=self._no_mono_if_stereo)
```

Comment:
max_chessboard_speed = self._max_chessboard_speed does not need whitespaces around =, because it is an argument
(same happens 4 lines below)